src/plugins/header/snippets/snippets.ts

```diff
@@ -148,7 +137,7 @@ export class UIPSnippets extends UIPPlugin {
     }
   }
 
-  @bind
+  @listen('click')
```

Comment:
It looks like it is the same listener that is used below. Possibly selector part was missed

Reply:
thank you! fixed :)